extensions/wikibase/module/langs/translation-en.json

```diff
@@ -109,6 +109,7 @@
     "wikibase-schema/incomplete-schema-could-not-be-saved": "Your schema is incomplete so it cannot be saved yet.",
     "wikibase-schema/unsaved-warning": "You have made unsaved changes to your Wikibase schema. Close anyway?",
     "wikibase-schema/incompatible-site-iri-reconciled-column": "This column is reconciled to the identifier space $1, but this input requires a column reconciled to $2.",
+    "wikibase-schema/expected-value": " with value set as ",
```

Comment:
It would be better to define a second version of the full message. Constructing localized messages by concatenating them can cause problems because you are assuming a specific word order which might not work in all languages.

Reply:
@wetneb The reason a separate label was added is because the value suggestion is optional. Any suggestion on how to handle this case if we were to include the message in the full message. Will it be easier if instead of verbose string we use something like  "Quebec ... (P633) requires statement with property country (P17) **[ Expected value - Canada ]"**

Reply:
Define two messages for the warning text:
* one for the full text of the warning description, without suggested value
* one for the full text of the warning description, with suggested value

Reply:
Added separate label entries for warnings where suggested value is available